package/src/components/TextInput/v1/TextInput.md

```diff
@@ -228,16 +222,18 @@ All of the icon props will accept a few different types of nodes to create the i
  - `<svg>` tag for inline SVG
  - Plain text
 
-Inline SVG icon
+Here is how to use an SVG icon to replace the default `iconValid`, `iconError`, `iconClear` and `iconClearAccessibilityText`:
 
 ```jsx
 const TwoColumnExamples = require("../../../../../styleguide/src/components/TwoColumnExamples").default;
 
-const svg = <svg version="1.1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 14 14" style={{ height: "100%" }}><path d="M9.926 9.105l-2.105-2.105 2.105-2.105-0.82-0.82-2.105 2.105-2.105-2.105-0.82 0.82 2.105 2.105-2.105 2.105 0.82 0.82 2.105-2.105 2.105 2.105zM7 1.176c3.227 0 5.824 2.598 5.824 5.824s-2.598 5.824-5.824 5.824-5.824-2.598-5.824-5.824 2.598-5.824 5.824-5.824z"></path></svg>;
+const smile = <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"  style={{ height: "100%", verticalAlign: "middle" }}> <path fill="currentColor" d="M248 8C111 8 0 119 0 256s111 248 248 248 248-111 248-248S385 8 248 8zm0 448c-110.3 0-200-89.7-200-200S137.7 56 248 56s200 89.7 200 200-89.7 200-200 200zm-80-216c17.7 0 32-14.3 32-32s-14.3-32-32-32-32 14.3-32 32 14.3 32 32 32zm160 0c17.7 0 32-14.3 32-32s-14.3-32-32-32-32 14.3-32 32 14.3 32 32 32zm4 72.6c-20.8 25-51.5 39.4-84 39.4s-63.2-14.3-84-39.4c-8.5-10.2-23.7-11.5-33.8-3.1-10.2 8.5-11.5 23.6-3.1 33.8 30 36 74.1 56.6 120.9 56.6s90.9-20.6 120.9-56.6c8.5-10.2 7.1-25.3-3.1-33.8-10.1-8.4-25.3-7.1-33.8 3.1z" /></svg>;
 
-<TwoColumnExamples hasDarkRightBackground>
-  <TextInput name="example" placeholder="Hint" icon={svg} />
-  <TextInput name="example" placeholder="Hint" icon={svg} isOnDarkBackground />
+const unhappy = <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"  style={{ height: "100%", verticalAlign: "middle" }}> <path fill="currentColor" d="M248 8C111 8 0 119 0 256s111 248 248 248 248-111 248-248S385 8 248 8zm0 448c-110.3 0-200-89.7-200-200S137.7 56 248 56s200 89.7 200 200-89.7 200-200 200zm-80-216c17.7 0 32-14.3 32-32s-14.3-32-32-32-32 14.3-32 32 14.3 32 32 32zm160-64c-17.7 0-32 14.3-32 32s14.3 32 32 32 32-14.3 32-32-14.3-32-32-32zm-80 128c-40.2 0-78 17.7-103.8 48.6-8.5 10.2-7.1 25.3 3.1 33.8 10.2 8.4 25.3 7.1 33.8-3.1 16.6-19.9 41-31.4 66.9-31.4s50.3 11.4 66.9 31.4c8.1 9.7 23.1 11.9 33.8 3.1 10.2-8.5 11.5-23.6 3.1-33.8C326 321.7 288.2 304 248 304z"/></svg>;
+
+<TwoColumnExamples hasDarkRightBackground hasDarkLeftBackground>
+  <TextInput name="example" placeholder="I am valid." iconClear={unhappy} iconValid={smile} hasBeenValidated isOnDarkBackground/>
+  <TextInput name="example" placeholder="I am invalid!" iconClear={unhappy} iconError={unhappy} iconValid={smile} iconClearAccessibilityText="Oops!" shouldAllowLineBreaks isOnDarkBackground errors={["error"]} />
```

Comment:
These need to be updated. All of the `icon*` props are within `components` prop now